microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,414 @@ ms.author: msedgedevrel
+
+Example:
+
+```
+Channel Search Kind: 1 (Least Stable)
+Release Channels: 0 (Stable), 1 (Beta)
+```
+
+This example setting causes the loader to try to run the app with Edge Beta if Edge Beta is installed on the device, and otherwise fall back to using the Stable WebView2 Runtime.
+
+The release channels can similarly be set through the API, registry key, environment variable, or group policy.
+
+
+##### [API](#tab/api)
+
+By default, the loader considers all of the release channels.  Set `ReleaseChannels` to a mask of the channels you would like the loader to search for.
+
+.NET:
+* [CoreWebView2EnvironmentOptions.ReleaseChannels Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.releasechannels)
+* [CoreWebView2ReleaseChannels Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2releasechannels)
+
+WinRT:
+* [CoreWebView2EnvironmentOptions.ReleaseChannels Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#releasechannels)
+* [CoreWebView2ChannelSearchKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2releasechannels)
+
+Win32:
+* [ICoreWebView2EnvironmentOptions7::get_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#get_releasechannels)
+* [ICoreWebView2EnvironmentOptions7::put_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#put_releasechannels)
+* [COREWEBVIEW2_RELEASE_CHANNELS enum](/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_release_channels)
+
+
+##### [Registry key](#tab/registry-key)
 
 1. Open a PowerShell terminal or PowerShell-enabled command prompt.
 
 1. Modify and then run the following command:
 
-   `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /v WebView2APISample.exe /t REG_SZ /d "1"`
+   ```reg
+   REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels /v WebView2APISample.exe /t REG_SZ /d "0,1"
+   ```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.  Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine, and can result in unexpected behavior.
+
+
+##### [Environment variable](#tab/environment-variable)
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+Name: `WEBVIEW2_RELEASE_CHANNELS`  <!-- 2 trailing spaces = linebreak -->
+Value: `<string of desired channels (ex: 0,1)>`
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+The environment variable, if set globally, is applied to all apps that use WebView2 on the machine.  In contrast, the registry key can be set per-app.
```

Comment:
Reword like elsewhere in this file?  Elsewhere in this file reads:
The target app can't be set when using the environment variable approach.  Therefore, when set as a global environment, this setting will affect all apps that use WebView2 on the machine.
@victorhuangwq

Reply:
```
The target app can't be set when using the environment variable approach.
Therefore, when set as a global environment, this setting will affect 
all apps that use WebView2 on the machine.
```